service/src/main/java/com/epam/esm/constant/CustomHttpStatus.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.esm.constant;
+
+/** The http response states interface. */
+public interface CustomHttpStatus {
```

Comment:
Interfaces are not intended for using as constants holders.

Reply:
Changed to class